engine/src/main/scripts/dtcli

```diff
@@ -67,7 +67,7 @@ fi
 
 if [ -f "$MVN_GENERATED_PATH" ]; then
   # development launch mode
-  DT_CORE_JAR="$BUILD_DIR/apex-engine.jar"
+  DT_CORE_JAR=`find "$BUILD_DIR" -maxdepth 1 -regex ".*\/apex-engine-.*[^tests]\.jar"`
```

Comment:
May be look at resource formatting. There is also another way to pass this kind of information through maven. If you can dig up jarpath source code, probably I had used something similar there.